nlp_primitives/tests/test_universal_sentence_encoder.py

```diff
@@ -13,3 +18,16 @@ def test_regular(universal_sentence_encoder):
     a = a.mean().round(7).astype('str')
     b = pd.Series(['-0.0007475', '0.0032088', '0.0018552', '0.0008256', '0.0028342'])
     assert a.equals(b)
+
+
+def test_without_tensorflow(universal_sentence_encoder):
+    # Simulate tensorflow being missing
+    tf_mod = sys.modules['tensorflow']
+    sys.modules['tensorflow'] = None
+    err_message = "In order to use the UniversalSentenceEncoder primitive install 'nlp_primitives[complete]'"
+    with pytest.raises(ImportError) as error:
+        UniversalSentenceEncoder()
+    assert error.value.args[0] == err_message
+
+    # Add tensorflow back to sys.modules
+    sys.modules['tensorflow'] = tf_mod
```

Comment:
Good catch. If something failed in the assertions, this would cause all downstream tests that need tensorflow to fail. I updated to move the change to `sys.modules` into a fixture as you suggested and that fixes the problem.